src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs

```diff
@@ -73,17 +71,16 @@ public class CommitmentOrchestrator : BaseCommitmentOrchestrator
                 throw new ArgumentNullException(nameof(uniqueUlnValidator));
             if (apprenticeshipMapper == null)
                 throw new ArgumentNullException(nameof(apprenticeshipMapper));
-
+            if (featureToggleService == null)
```

Comment:
It's probably not worthwhile putting in these checks. Structure Map (as I understand it), won't instantiate a class if the dependency isn't resoved

Reply:
I've confirmed that's the case - StructureMap throws a StructureMapConfigurationException. I'll remove the null checks.